perfmetrics/scripts/continuous_test/gke/orbax_benchmark/run_benchmark.py

```diff
@@ -392,7 +390,7 @@ async def main():
     with tempfile.TemporaryDirectory() as temp_dir:
         try:
             setup_task = asyncio.create_task(setup_gke_cluster(args.project_id, args.zone, args.cluster_name, args.network_name, args.subnet_name, args.zone.rsplit('-', 1)[0], args.machine_type, args.node_pool_name))
-            build_task = asyncio.create_task(build_gcsfuse_image(args.project_id, args.gcsfuse_branch, temp_dir))
+            build_task = asyncio.create_task(build_gcsfuse_image(args.gcsfuse_branch, temp_dir))
```

Comment:
project ID is not required anymore? If it is taking from gcloud env then please write it up in README.